Mopy/bash/brec.py

```diff
@@ -136,14 +139,179 @@ def pack(self):
         raise exception.AbstractError
 
 #------------------------------------------------------------------------------
+class NewRecordHeader(BaseRecordHeader):
+    """
+    TODO: add a method to return a tuple of the block coordinates. These
+    coordinates should not be packed and unpacked aside from a separate method
+    to avoid human error and reverse them making X and Y become Y and X by
+    mistake.
+
+    Pack or unpack the record's header.
+    """
+
+    # size = NewRecordHeader.get_magic_number()
+
+    pack_formats = {
```

Comment:
This should go back to game init.py files - this will greatly simplify code and avoid having to import brec in main which for highly technical reasons #360 is a nono. When game handling is refactored code duplication will disapear but for now we have to have one of those for oblivion and the same for rest of games - so for rest of games `pack_formats = { 'RECORD': '=4s5I', 'GRUP0' : '=4sI4s3I', etc}`

Reply:
Notice now we dropped the tuples and this makes code much more readable